sleap/gui/web.py

```diff
@@ -146,6 +149,49 @@ def get_release(self, version: str) -> Release:
         )
 
 
+@attr.s(auto_attribs=True)
+class AnnouncementChecker:
+    """Checker for new announcements on the bulletin page of sleap."""
+
+    bulletin_json_path: str = BULLETIN_JSON
+    previous_announcement_date: str = prefs.prefs["announcement last seen date"]
+
+    def check_for_announcement(self) -> bool:
+        """Returns if new announcements are available."""
+        try:
+            # Attempt to open the file in read mode
+            with open(self.bulletin_json_path, 'r', encoding='utf-8') as jsf:
+                # Load the JSON data into a Python data structure
+                data = json.load(jsf)
+                latest_data = data[0]
+                
+                if latest_data['date'] != self.previous_announcement_date:
+                    return True
```

Comment:
Can we instead convert both the `latest_date['date']` and `self.previous_announcement_date` to a format that allows `>`, `<` comparison? I see this erroring when the `self.previous_announcement_date` > `latest_date['date']` and still showing the announcement although it was already seen.